app/src/main/javascript/bundles/experiment-page/src/ExperimentPageRouter.js

```diff
@@ -8,132 +8,218 @@ import TSnePlotViewRoute from './TSnePlotViewRoute'
+    routeProps: PropTypes.shape(RoutePropTypes)
 }
 
 
-const TabContent = ({type, tabProps, commonProps, routeProps}) => {
-  // Pass in the search from location
-  const Tab = tabTypeComponent[type]
+const TabContent = ({type, tabProps, commonProps, routeProps, resultTabView}) => {
+    // Pass in the search from location
+    const Tab = tabTypeComponent[type]
 
-  return (
-    Tab ? <Tab {...tabProps} {...commonProps} {...routeProps}/> : null
-  )
+    return (
+        Tab ? <Tab {...tabProps} {...commonProps} {...routeProps} enableView={resultTabView} /> : null
+    )
 }
 
 TabContent.propTypes = {
-  type: PropTypes.string.isRequired,
-  tabProps: PropTypes.object,
-  commonProps: PropTypes.shape(TabCommonPropTypes),
-  routeProps: PropTypes.shape(RoutePropTypes)
+    type: PropTypes.string.isRequired,
+    tabProps: PropTypes.object,
+    commonProps: PropTypes.shape(TabCommonPropTypes),
+    routeProps: PropTypes.shape(RoutePropTypes),
+    resultTabView : PropTypes.func
 }
 
 const RedirectWithSearchAndHash = (props) =>
-  <Redirect to={{ pathname: props.pathname, search: props.location.search, hash: props.location.hash}} />
+    <Redirect to={{ pathname: props.pathname, search: props.location.search, hash: props.location.hash}} />
 
 RedirectWithSearchAndHash.propTypes = {
-  pathname: PropTypes.string.isRequired,
-  location: PropTypes.shape({
-    search: PropTypes.string.isRequired,
-    hash: PropTypes.string.isRequired
-  }).isRequired
+    pathname: PropTypes.string.isRequired,
+    location: PropTypes.shape({
+        search: PropTypes.string.isRequired,
+        hash: PropTypes.string.isRequired
+    }).isRequired
 }
 
 const RedirectWithLocation = withRouter(RedirectWithSearchAndHash)
 
 const ExperimentPageRouter = ({atlasUrl, resourcesUrl, experimentAccession, species, accessKey, tabs}) => {
-  const tabCommonProps = {
-    atlasUrl,
-    resourcesUrl,
-    experimentAccession,
-    species,
-    accessKey
-  }
-
-  return (
-    <BrowserRouter
-      basename={URI(`experiments/${experimentAccession}`, URI(atlasUrl).path()).toString()}>
-      <div>
-        <Route
-          path={`/`}
-          render={
-            (routeProps) =>
-              <TopRibbon
-                tabs={tabs}
-                routeProps={routeProps} />
-          } />
-        <Switch>
-          {
-            tabs.map((tab) =>
-              <Route
-                key={tab.type}
-                path={`/${tab.type}`}
-                render={
-                  (routeProps) =>
-                    <TabContent
-                      type={tab.type}
-                      tabProps={tab.props}
-                      commonProps={tabCommonProps}
-                      routeProps={routeProps} />
-                } />
-            )
-          }
-          <RedirectWithLocation pathname={`/${tabs[0].type}`} />
-        </Switch>
-      </div>
-    </BrowserRouter>
-  )
+
+    const [enableResultTab, setEnableResultTab] = useState(false)
+
+
```

Comment:
Extra empty lines. Please check your codes and make it as clean as possible, I also noticed, you've changed some original codes indention. Please double check the commands you've modified, maybe it's easier to be observed in github.

Reply:
Removed